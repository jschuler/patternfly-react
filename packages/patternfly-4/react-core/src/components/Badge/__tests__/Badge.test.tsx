// Auto-generated do not edit

import * as React from 'react';
import { mount } from 'enzyme';
import { Badge } from '../Badge';

describe('Badge test', () => {
  it('Badge should match snapshot', () => {
    const view = mount(
      <Badge
        isRead={false}
				children={''}
				className={"''"}
      />);
    expect(view).toMatchSnapshot();
  });
});
