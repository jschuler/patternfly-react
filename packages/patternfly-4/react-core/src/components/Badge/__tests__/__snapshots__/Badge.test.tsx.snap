// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge test Badge should match snapshot 1`] = `
<Component
  className="''"
  isRead={false}
>
  <span
    className="pf-c-badge pf-m-unread ''"
  />
</Component>
`;
